internal/provider/const.go

```diff
@@ -51,4 +51,6 @@ const (
 	PrimaryAccountIdKey = "primary_account_id"
 	ScopeKey            = "scope"
 	ParentScopeIdKey    = "parent_scope_id"
+
+	GroupMemberIdsKey = "member_ids"
```

Comment:
We generally like to add comments to exported variables so godocs has a comment for it. I see we did not do that for some of the above consts, but would be good to add it here